static/Functions.js

```diff
@@ -3,16 +3,17 @@
 
 
 // The initialize function is required for all add-ins.
-Office.initialize = function () {
-  jQuery(document).ready(function () {
+Office.initialize = function() {
+  jQuery(document).ready(function() {
     console.log('JQuery initialized');
   });
 };
 
 console.log('loading supply-chain add-in');
 
 // TODO move to configuration retrieved from the server
-const containerName = "attachments";
+const USER_TOKEN_HEADER_KEY = 'user-token';
+const CONTAINER_NAME = "attachments";
```

Comment:
do we even use this CONTAINER_NAME?
I see that we add this to the data, but I don't think we use it on the server side.
just make sure and remove.